### DISCUSSION TITLE
When using with wildcard, dvr-scan detects motion across two videos

### DISCUSSION THREAD

**Discussion by nloui** [Breakthrough/DVR-Scan#135] [General]
When running dvr-scan on a folder of videos:

```
dvr-scan -i W:\*.mp4 -d events/ -b MOG2_CUDA --threshold 0.25 -bb
```

My events folder includes events that appear to be "detecting motion" by merging two clips.  

Here's an example: https://www.dropbox.com/s/charmo8khq6b0xl/MP4_1580728320.DSME_0011.avi?dl=0

**Comment by Breakthrough** [comment]
All inputs to DVR-Scan are concatenated together.  This is typically as people want to run the software on DVR footage which ends up being split into chunks, but the resulting motion events are contiguous across each file.

If this is undesirable for your use case, the preferred approach is to write a for-loop to run DVR-Scan on each file that matches the path `W:\*.mp4`.  This comes up from time to time, so I may consider adding a flag for this purpose (e.g. `-n`/`--no-concatenate`). Thoughts?
